components/financial_assistance/app/views/financial_assistance/shared/_consumer_fields.html.erb

```diff
@@ -1,5 +1,6 @@
 <% if @bs4 %>
   <div id="consumer_fields" class="focus_effect">
+    <h2 class="mt-4"><%= l10n("insured.consumer_roles.demo_info_for") %> <%= @applicant.full_name %></h2>
```

Comment:
this line needs a check to make sure the applicant is an applicant so it doesn't break the new form. something like below (but with translation)
    <h2 class="mt-4"><%= l10n("insured.consumer_roles.demo_info_for") %> <%= @applicant.class == FinancialAssistance::Applicant ? @applicant.full_name : "this person" %></h2>

Reply:
Interestingly, the consumer fields partial used on new comes from the main app, `shared/consumer_fields" rendered by `dependent_form`, so this isn't quite a problem. I added this line for robustness, anyway